lib/transaction.js

```diff
@@ -11,6 +11,27 @@ var Transaction = require('loopback-connector').Transaction;
 
 module.exports = mixinTransaction;
 
+var mapIsolationLevel = function(isolationLevelString) {
+  var ret = 2;
+  switch (isolationLevelString) {
+    case Transaction.READ_UNCOMMITTED:
+      ret = 1;
+      break;
+    case Transaction.SERIALIZABLE:
+      ret = 4;
+      break;
+    case Transaction.REPEATABLE_READ:
+      ret = 8;
+      break;
+    case Transaction.READ_COMMITTED:
```

Comment:
Yes... However without `case Transaction.READ_COMMITTED:` you would get the your desired result as well since it anyway falls through `default ` and you would have less code in this case... On the other hand, one may debate that your code make it more clear that `Transaction.READ_COMMITTED` intentionally falls through `default`...
Anyway I'm fine with either way...